### PR TITLE
Update eu-residency.md

### PR DESCRIPTION
**Pull request by anaerquicia**
Syntax corrections

### CODE REVIEW

File: pages/docs/privacy/eu-residency.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ By default Mixpanel stores user data on it's US Servers via the Google Cloud Pla
...
+You can find where your data is stored under Project Settings > Data Residency for existing projects. Additionally, projects stored in the EU will have a URL that starts with 'eu.mixpanel.com', whereas projects stored in the US will have a 'mixpanel.com' URL. If the wrong Data Residency location was chosen before implementation, you will need to create a new project with the applicable data storage option and migrate all your existing data. Mixpanel cannot assist with migrating an existing project with the wrong residency location. You can find out more about creating a new project [here](https://docs.mixpanel.com/docs/orgs-and-projects/managing-projects#creating-projects).
```
Comment (mavlee) [inline pages/docs/privacy/eu-residency.md]: IMO the first sentence was better before

Comment (mavlee) [reply pages/docs/privacy/eu-residency.md]: the link should be using a relative link

/docs/orgs-and-projects........

**Comment by myronkaifung** [comment]
Hi @anaerquicia , would you please review the comments?

File: pages/docs/privacy/eu-residency.md (hunk 1 above)
Comment (anaerquicia) [reply pages/docs/privacy/eu-residency.md]: Hi @mavlee. Sorry about that, I missed this notification.

I've now corrected it, could you review it and let me know if it's OK?

**Review by mavlee — APPROVED** [review]

**Pull request merged**